src/app/components/SeriesCard/SeriesCard.module.css

```
@@ -0,0 +1,31 @@
+.container {
+  background: transparent;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 0.8em;
+  font-size: 3vw;
+}
+
+.img {
+  width: 100%;
+  height: auto;
+  grid-column: 1/2;
+}
+
+.seriesInfo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.5em;
+}
+.headline {
```

This is not exactly BEM like. 
Maybe this is better?
```suggestion
.seriesInfo__headline {
```

---

ok